CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
+
 cmake_minimum_required(VERSION 2.8.6)
 
 ## The path to the FindEigen file.
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3")
```

Comment:
I'm pretty sure this line isn't necessary. Try creating an empty directory and putting a CMakeLists.txt in it containing:

    cmake_minimum_required(VERSION 2.8.6)

    message("${CMAKE_CXX_FLAGS_RELEASE}")

and then running "cmake ."